### ISSUE TITLE
Add active item to navigation

### ISSUE THREAD

**Issue by yemisi-o** [she-code-africa/SCA-Website#53]
This is to make the hover state of navigation items stay fixed as the active state for any page in view i.e. When user is on any current page in view, the nav should have it in active state like this:

![image](https://user-images.githubusercontent.com/17818475/69934795-51211900-14d3-11ea-9996-cccc7167c53c.png)

Also, the 'Home' should not be visible on the navbar when user is on the home page, it should only be visible on the nav when user is on other pages of the website

**Comment by mazma1** [comment]
@yemisi please what's wrong with having the "Home" link visible while on the home page?

Since this is on the navbar, does it make sense to have the "Sign In" link since most users won't have any need for it? I feel it may be a source of confusion and I actually thought of taking it out

**Comment by yemisi-o** [comment]
@mazma just seeing this comment after approving your PR.
**Home link**: There is nothing wrong in having it, it is mostly to reduce nav clutter on the main landing page but it is still to be included on the other pages so it is easy to navigate back to home for users who aren't used to clicking on logos to navigate to the homepage. For other pages, the link/nav item will still be shown on their corresponding pages for feedback (i.e. in conjunction with the active state, the user can easily know what page they are on).
**Sign In Link**: I agree and I have included it in #51 as the things to remove. It was added to the original design for the forum which we are not implementing yet, it doesn't make sense to keep it there if it is not functional.

In summary, having or not having the 'Home' link on the homepage is trivial and can be skipped: if we have more space on the nav after removing the items mentioned in #51 and/or if the effort to implement it isn't minimal. Thanks for asking
